src/panels/lovelace/cards/hui-thermostat-card.ts

```diff
@@ -101,6 +104,14 @@ export class HuiThermostatCard extends hassLocalizeLitMixin(LitElement)
     this._config = { theme: "default", ...config };
   }
 
+  public connectedCallback(): void {
+    super.connectedCallback();
+    this._attached = true;
+    if (this._updated && !this._loaded) {
```

Comment:
This check and the one in `firstUpdated` can be combined and placed in `_initialLoad`. Not sure which is clearer.

Reply:
I prefer to only call `firstUpdated` once

Reply:
Fair enough.